src/main/java/conversion/Individual.java

```diff
@@ -28,14 +28,14 @@
  * Represents a file conversion request to the server. Allows storage of UUID's
  * for identification of clients which are requesting file conversions.
  */
-class Individual {
+public class Individual {
 
     public final String uuid;
     public boolean isAlive = true;
-    public String outputDir = null;
     final long timestamp;
     public String state;
     public String errorCode;
+    public Object data;
```

Comment:
I'm not convinced that using a public object is the best approach to handle this class with. 

I propose that we adjust this to one of the following:

- We add the token into the customValues map
  - Would require adding a getValue method to the class
  - Would mean that get requests to the server will return the token value (due to toJsonString)
- We add the token into the customValues map and alter the toJsonString method
  - Similar to the first suggestion but adds the capability to hide the custom values and have it set to be hidden by default.
- We change 'data' to a HashMap<String, String> called hiddenValues
  - The purpose would be similar to customValues but it would not be returned in toJsonString()

@leonatherton - Thoughts?

Reply:
@daniel-warren The purpose of the value is that is allows unspecified data to be attached the Individual. It could be something like a String or Map value, or it could be more complex like a Custom class that base-conversion-servlet doesn't know about.

I don't think reusing the customValues map would be very tidy as there would need to be additional logic to handle which values should be returned in toJsonString().

The object comes in via request.getAttribute("data") which returns an Object, so if we were to store the values in a Map<String, String> then we would be limiting what can be stored.

In our case we're probably going to want to store the token and whether BuildVu-SVG or BuildVu-HTML should be used for the conversion.

I made it a public field because the others fields are accessed in the same way, but I'd be happy to make it private with a getter/setter (should we do the same with the others?). We could also give it a more descriptive name - customPrivateData?

Reply:
@leonatherton I had missed that getAttribute("data") returns an Object, in that case I would have to agree that my proposed solutions are not fitting. I think the name customData sounds good and if we could have it private with a getter/setting that would be preferable. 

I would not be against encapsulating the other fields, however I believe this is out of scope for the current change and requires more work to update and document than the potential benefits.

Reply:
@daniel-warren I'm breaking compatibility with buildvu-microservice-example with the individual.outputDir field anyway, so it's probably a good time to change them. Is there any documentation that would need updating that you're aware of?

Reply:
@leonatherton - I had thought the README.md of BuildVu Microservice Example mentions it however I stand corrected. Other than updating where individual is used in code changing them all sounds good.